File: problem1.py
## problem 1

sum = 0
i: int = 0

while i <= 1000:
    i += 1
    if i%3 == 0:
        print("add", i)
        sum += i
    elif i%5 == 0:
        print("add", i)
        sum += i

print(sum)

File: Miron/problem_1/problem_1.cpp
#include <iostream>

int main(){
    int n_max = 1000;
    int sum = 0;
    for (int i = 0; i < n_max; i++){
        if (i%3 == 0 || i%5 == 0){
            sum += i;
        }
    }
    std::cout << "Sum of all natural numbers up to " << n_max << " is " << sum << '\n';
}
